lib/ui/auth_flow/info_screen.dart

```diff
@@ -29,13 +35,18 @@ class InfoScreen extends StatelessWidget {
                   title,
                   style: const TextStyle(
                     fontSize: 16,
-                    fontWeight: FontWeight.bold,
+                    fontWeight: FontWeight.w700,
+                    color: Colors.black,
                   ),
                 ),
-                const SizedBox(height: 4),
+                const SizedBox(height: 6),
                 Text(
                   subtitle,
-                  style: TextStyle(fontSize: 15, color: AppColors.glitch400),
+                  style: TextStyle(
+                    fontSize: 14,
+                    color: Colors.grey.shade700,
```

Comment:
Let's use colors from our AppColors class (or add if absent), so we don't have to revisit them later.